models/tests.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2016 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package models
```

Comment:
I think this file should be named `something_test.go`, so it whould not be compiled if not running tests.